tests/Core/Ruleset/PopulateTokenListenersNamingConventionsTest.php

```diff
@@ -11,70 +11,58 @@
-use PHPUnit\Framework\TestCase;
+use PHP_CodeSniffer\Tests\Core\Ruleset\AbstractRulesetTestCase;
 
 /**
  * Test handling of sniffs not following the PHPCS naming conventions in the Ruleset::populateTokenListeners() method.
  *
  * @covers \PHP_CodeSniffer\Ruleset::populateTokenListeners
  */
-final class PopulateTokenListenersNamingConventionsTest extends TestCase
+final class PopulateTokenListenersNamingConventionsTest extends AbstractRulesetTestCase
 {
 
 
     /**
      * Verify a warning is shown for sniffs not complying with the PHPCS naming conventions.
      *
      * Including sniffs which do not comply with the PHPCS naming conventions is soft deprecated since
-     * PHPCS 3.12.0, hard deprecated since PHPCS 3.13.0 and support will be removed in PHPCS 4.0.0.
+     * PHPCS 3.12.0, hard deprecated since PHPCS 3.13.0 and support has been removed in PHPCS 4.0.0.
      *
      * @return void
      */
     public function testBrokenNamingConventions()
     {
+        $expectedMessage  = 'ERROR: The sniff BrokenNamingConventions\\Sniffs\\MissingCategoryDirSniff does not comply';
+        $expectedMessage .= ' with the PHP_CodeSniffer naming conventions.'.PHP_EOL;
+        $expectedMessage .= 'Contact the sniff author to fix the sniff.'.PHP_EOL;
+        $expectedMessage .= 'ERROR: The sniff NoNamespaceSniff does not comply with the PHP_CodeSniffer naming conventions.'.PHP_EOL;
+        $expectedMessage .= 'Contact the sniff author to fix the sniff.'.PHP_EOL;
+        $expectedMessage .= 'ERROR: The sniff Sniffs\PartialNamespaceSniff does not comply with the PHP_CodeSniffer naming conventions.'.PHP_EOL;
+        $expectedMessage .= 'Contact the sniff author to fix the sniff.'.PHP_EOL;
+        $expectedMessage .= 'ERROR: The sniff BrokenNamingConventions\Sniffs\Category\Sniff does not comply';
+        $expectedMessage .= ' with the PHP_CodeSniffer naming conventions.'.PHP_EOL;
+        $expectedMessage .= 'Contact the sniff author to fix the sniff.'.PHP_EOL;
+        $expectedMessage .= 'ERROR: The sniff BrokenNamingConventions\Sniffs\Sniffs\CategoryCalledSniffsSniff does not';
+        $expectedMessage .= ' comply with the PHP_CodeSniffer naming conventions.'.PHP_EOL;
+        $expectedMessage .= 'Contact the sniff author to fix the sniff.'.PHP_EOL;
+        $expectedMessage .= 'ERROR: The sniff BrokenNamingConventions\Sniffs\Category\SubDir\TooDeeplyNestedSniff';
+        $expectedMessage .= ' does not comply with the PHP_CodeSniffer naming conventions.'.PHP_EOL;
+        $expectedMessage .= 'Contact the sniff author to fix the sniff.'.PHP_EOL.PHP_EOL;
+
+        $this->expectRuntimeExceptionMessage($expectedMessage);
+
         // Set up the ruleset.
         $standard = __DIR__.'/PopulateTokenListenersNamingConventionsTest.xml';
         $config   = new ConfigDouble(["--standard=$standard"]);
         $ruleset  = new Ruleset($config);
 
         // The "Generic.PHP.BacktickOperator" sniff is the only valid sniff.
```

Comment:
I wonder if this code comment is still needed now that the other sniff codes were removed?

Reply:
I don't think it does any harm and it kind of explains why the assertion after it is being run, so I think I'll leave it for now.